### PR TITLE
gui: don't allow cpu with fma support disable accurate path

### PR DESCRIPTION
**Pull request by illusion0001**
~~#7930~~ now merged.
Suggestions for description are welcome.

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -963,8 +963,16 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->debugConsoleMode, emu_settings::DebugConsoleMode);
 	SubscribeTooltip(ui->debugConsoleMode, tooltips.settings.debug_console_mode);
 
-	m_emu_settings->EnhanceCheckBox(ui->accuratePPUfma, emu_settings::AccuratePPUfma);
-	SubscribeTooltip(ui->accuratePPUfma, tooltips.settings.accurate_ppu_fma);
+	if (utils::has_fma3() || utils::has_fma4())
+	{
+	ui->accuratePPUfma->setEnabled(false);
+	SubscribeTooltip(ui->accuratePPUfma, tr("Improves FMA operations' accuracy at the cost of performance.\nWhile disabling it might give a decent performance boost, it may also introduce subtle bugs that otherwise do not occur."));
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:969]: indentation.

If there's a tooltip class.... why don't you use it?

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp:969]: and why is there a long tooltip for a disabled checkbox?

Comment (illusion0001) [reply rpcs3/rpcs3qt/settings_dialog.cpp:969]: Copypasta 🤦‍♀️ will fix.

Comment (illusion0001) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Should be resolved now. Please review.

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 2]
```diff
@@ -963,8 +963,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->debugConsoleMode, emu_settings::DebugConsoleMode);
 	SubscribeTooltip(ui->debugConsoleMode, tooltips.settings.debug_console_mode);
 
-	m_emu_settings->EnhanceCheckBox(ui->accuratePPUfma, emu_settings::AccuratePPUfma);
-	SubscribeTooltip(ui->accuratePPUfma, tooltips.settings.accurate_ppu_fma);
+	if (utils::has_fma3() || utils::has_fma4())
+	{
+	ui->accuratePPUfma->setEnabled(false);
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:968]: ???

Comment (illusion0001) [reply rpcs3/rpcs3qt/settings_dialog.cpp:968]: What's the confusion?

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp:968]: it's still not indented

Comment (illusion0001) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Is it good now?

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: It's worse...
Idk what you're doing... But it's not working

Comment (illusion0001) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: *Should* be good now?

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Yes. Looks good now

**Comment by illusion0001** [comment]
Rebased. Ready for review.

**Comment by psennermann** [comment]
when greyed out, shouldn't the option show the flag? Because without the flag visually it seems that the option is disabled and cannot be enabled...

**Comment by dio-gh** [comment]
He tried it but couldn't figure out how to make it look like that (he told me on Discord about this). Fair point though.

**Comment by illusion0001** [comment]
@psennermann @RainbowCookie32 addressed it in #8001

**Pull request merged**